Shared/Samples/Monitor changes to layer view state/MonitorChangesToLayerViewStateView.swift

```diff
@@ -37,27 +37,28 @@ struct MonitorChangesToLayerViewStateView: View {
         return featureLayer
     }()
     
-    /// The text representing the view status of the feature layer.
-    @State private var statusText = ""
-    
     /// A Boolean value indicating whether the feature layer is visible.
     @State private var layerIsVisible = true
+
+    /// The current `LayerViewState.Status`for the view.
+    @State private var layerStatus: LayerViewState.Status = .loading
```

Comment:
Should the initial value be an empty set so that the status is "Unknown" until updated?

```suggestion
    @State private var layerStatus: LayerViewState.Status = []
```